content/en/docs/installation/kubernetes.md

```diff
@@ -46,9 +46,16 @@ are included in a single YAML manifest file:
 
 Install the `CustomResourceDefinitions` and cert-manager itself
 ```bash
-$ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.13.1/cert-manager.yaml
+# Kubernetes 1.15+
+$ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.14.0/cert-manager.yaml
```

Comment:
These need to point at the alpha tag, or otherwise we need to keep them held back for now in a separate `release-0.14` branch 😄

Reply:
We should put these on a seperate release branch, we don't want to put the alpha version on our homepage yet

Reply:
:+1: we now have the `release-0.14` branch - shall we keep this as-is and then make it live alongside v0.14.0? That SGTM, it's just a change from what we've done before!

Reply:
With https://github.com/cert-manager/website/pull/147, it may make sense to change this to `alpha` come to think.. it means we can make these docs available ahead of time and they will still be valid 😄